extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -298,6 +298,11 @@ oppia.directive('graphViz', [
             $scope.state.currentMode = null;
           });
 
+          $scope.isMobile = false;
```

Comment:
I think it's fine for this to be a local variable and not attach it to $scope?

Reply:
Actually, I had used $scope to print some help text on screen and display that based on isMobile.